store/store.go

```diff
@@ -340,10 +332,10 @@ func (d MongoStoreClient) HasMedtronicLoopDataAfter(userID string, date string)
 	defer session.Close()
 
 	query := bson.M{
-		"_active":        true,
-		"_userId":        userID,
-		"_schemaVersion": bson.M{"$gt": 0},
-		"time":           bson.M{"$gte": date},
+		"_active":                            true,
+		"_userId":                            userID,
+		"_schemaVersion":                     bson.M{"$gt": 0},
```

Comment:
Should this use the input parameters limiting the schema version?

Reply:
`_schemaVersion` has always been internal. I don't even see why it's particularly useful for us. Maybe we should check what it's function is and remove it altogether?
Either way, I'd make a separate card for that. This PR doesn't change the behaviour at all.

Reply:
Just saw this in my GitHub feed. We can remove `_schemaVersion` as it really isn't used anymore. *However*, we need to delete all `_schemaVersion=0` from the database before we remove it. The `_schemaVersion=0` is from data prior to a "reset" and is, essentially, invalid.